### ISSUE TITLE
Update system for space cloud

### ISSUE THREAD

**Issue by 10allday** [spaceuptech/space-cloud#1417]
### The problem faced currently?

Everytime a new update comes out, you have to manually update the software.

### How can we solve it?

Have an update system, that can do the job for you.

>If you want this feature to be implemented, give it a thumbs up reaction, so that we can determine which features are important to you.
>👍

**Comment by YourTechBud** [comment]
Would using a command like `space-cli update` work?

**Comment by 10allday** [comment]
Yeah, also could have a button in the software for more ease of use.

**Comment by 10allday** [comment]
> Would using a command like `space-cli update` work?
Yeah, also could have a button in the software for more ease of use.

**Comment by 10allday** [comment]
> Would using a command like `space-cli update` work?

Yeah, also could have a button in the software for more ease of use.

**Comment by YourTechBud** [comment]
Makes sense
